### PR TITLE
Adapters should only update Path and Context

### PR DESCRIPTION
**Pull request by haiodo**
Adapters should only update Path and Context

Signed-off-by: Andrey Sobolev <haiodo@gmail.com>

### CODE REVIEW

File: pkg/networkservice/core/adapters/server_to_client.go [hunk 1]
```diff
@@ -44,9 +44,14 @@ func (s *serverToClient) Request(ctx context.Context, in *networkservice.Network
...
 	}
 	if in == nil {
-		in = &networkservice.NetworkServiceRequest{}
+		in = &networkservice.NetworkServiceRequest{
+			Connection: &networkservice.Connection{},
+		}
+	}
+	if conn != nil {
+		in.Connection.Context = conn.Context
+		in.Connection.Path = conn.Path
```
Comment (denis-tingajkin) [inline pkg/networkservice/core/adapters/server_to_client.go:53]: @haiodo What do you think about this simple patch?
```suggestion
		in = &networkservice.NetworkServiceRequest{}
	}
	if conn != nil {
	    if in.Connection == nil {
	        in.Connection = conn
	    } else {
		in.Connection.Context = conn.Context
		in.Connection.Path = conn.Path
	     }
```

**Comment by denis-tingajkin** [comment]
@haiodo Can you please resolve conflicts here?

**Comment by edwarnicke** [comment]
@haiodo Can this be closed?

**Comment by denis-tingaikin** [comment]
Closing due to outdated

**Pull request closed without merging**